### PR TITLE
Update Map Function parameters under models

### LINKED ISSUE #69: Set MapAction zoom parameter as default under model
https://github.com/fossasia/susi_api_wrapper/blob/aa934bc12cf68369193421e976bae7a340892d3e/python_wrapper/susi_python/models.py#L98

Currently while other map functions have zoom parameter as default without the user not to be bothered about the zoom value, here we have to supply the zoom parameter.

### PR DESCRIPTION
**Pull request by prateekiiest**
Fixes #69

### CODE REVIEW

**Comment by prateekiiest** [comment]
@mariobehling @Orbiter @stealthanthrax @hongquan @betterclever please review

File: python_wrapper/susi_python/models.py [hunk 1]
```diff
@@ -96,11 +96,14 @@ def __init__(self, columns):
 
 
 class MapAction(BaseAction):
-    def __init__(self, latitude, longitude, zoom):
+    def __init__(self, latitude, longitude, zoom= None):
```
Comment (hongquan) [inline python_wrapper/susi_python/models.py]: Check Python coding style (PEP 8).

**Review by hongquan — CHANGES_REQUESTED** [review]
Check Python coding style.

File: python_wrapper/susi_python/models.py (hunk 1 above)
Comment (prateekiiest) [reply python_wrapper/susi_python/models.py]: updated @hongquan

**Review by sansyrox — APPROVED** [review]

File: python_wrapper/susi_python/models.py (hunk 1 above)
Comment (hongquan) [reply python_wrapper/susi_python/models.py]: Still not correct: https://www.python.org/dev/peps/pep-0008/#other-recommendations

**Review by hongquan — APPROVED** [review]

**Pull request merged**